votepy/chamberlin-courant.py

```diff
@@ -0,0 +1,62 @@
+from ordinal_election import OrdinalElection
+from typing import Union, Iterable, List
+from generic_greed import greedy
+
+
+def chamberlin_courant_greedy(voting: Union[OrdinalElection, List[int]], size_of_committee: int,
+                              number_of_scored_candidates: int) -> List[int]:
+    """Greedy implementation of the chamberlin-courant rule
+    Args:
+        voting (Union[OrdinalElection, list[int]]): Voting for which the function calculates the committee
+        size_of_committee (int): Size of the committee
+        number_of_scored_candidates (int): Number of scored candidartes using k-borda rule
+    Returns:
+        list[int]: List of chosen candidates
+    """
+
+    def scoring_function(committee: Iterable[int], voting: OrdinalElection, candidate: int,
+                         number_of_scored_candidates: int):
+        committee = set(committee + [candidate])
```

Comment:
committee is an Iterable and for most Iterables adding a list won't work. 
```suggestion
        committee = set(committee) + {candidate}
```